includes/platform-checkout-user/class-platform-checkout-save-user.php

```diff
@@ -76,4 +83,17 @@ public function maybe_add_userdata_to_metadata( $metadata, $order ) {
 
 		return $metadata;
 	}
+
+	/**
+	 * Clears if platform checkout user data is set.
+	 *
+	 * @return void
+	 */
+	public function maybe_clear_session_key() {
+		$session_data = WC()->session->get( 'platform-checkout-user-data' );
```

Comment:
```suggestion
		$session_data = WC()->session->get( Platform_Checkout_Extension::PLATMORM_CHECKOUT_SESSION_KEY );
```
Why not use the lovely constant that you created over here?